driver-core/src/main/com/mongodb/client/model/expressions/Expression.java

```diff
@@ -166,6 +165,7 @@ public interface Expression {
      * {@code this} is a number, or the {@code other} number value if
      * {@code this} is null, or is missing, or is of any other non-number type.
      *
+     * @mongodb.server.release 4.4
```

Comment:
Moved this from https://github.com/mongodb/mongo-java-driver/pull/1032: We also need @since on the package and the new public classes/interfaces to express the driver version in which they were added.